apps/web/src/schema/cost-analysis/budget-usage/api-verbs/analyze.ts

```diff
@@ -5,12 +5,13 @@ export interface BudgetUsageAnalyzeResult {
     total_spent: number;
     name?: string;
     project_id?: string;
-    project_group_id?: string;
+    workspace_id?: string;
     data_source_id?: string;
     provider_filter?: {
         state?: string;
         providers?: string[];
     };
+    permission_group: 'WORKSPACE' | 'DOMAIN';
```

Comment:
nitpick: We have [PermissionGroup](https://github.com/cloudforet-io/console/blob/aaa2b1487668fc0b5e517e782edd188561813010/apps/web/src/schema/identity/role-binding/type.ts#L1C1-L1C66) type!!

Reply:
```suggestion
Pick<PermissionGroup, 'WORKSPACE' | 'DOMAIN'>
```